### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by Sheen-n29**
@GuidoPerren
@Danilcool

### CODE REVIEW

**Comment by Sheen-n29** [comment]
@Bryan24601

File: prac_06/car.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.name = name
+        self.fuel = fuel
+        self._odometer = 0
+
+    def __str__(self):
+        return f"{self.name} fuel={self.fuel} odometer={self._odometer}"
```
Comment (GuidoPerren) [inline prac_06/car.py:17]: Perfectly done.

File: prac_06/guitar.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    def get_age(self):
+        age = 2023 - self.year
+        return age
+
+    def is_vintage(self):
+        age = self.get_age()
+        if age >= 50:
+            return True
+        else:
+            return False
```
Comment (GuidoPerren) [inline prac_06/guitar.py:20]: if the usage of get_age is only to determine if the guitar is vintage, you could combine it into 1 function or even check for it within the constructor. I wouldn't consider this method more elegant, therefore your solution is still on par.

File: prac_06/guitars.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+def main():
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: "))
+        print(f"{name} ({year}) : ${cost:.2f} added")
```
Comment (GuidoPerren) [inline prac_06/guitars.py:11]: perfectly done. for aesthetics you could add a new line for better readability

**Review by GuidoPerren — APPROVED** [review]
Perfectly done.

**Pull request merged**